pom.xml

```diff
@@ -1115,6 +1119,15 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
```

Comment:
what's reason for adding this repo?

Reply:
Removed. Unused repository from earlier feature development work